bin/infrasim-init

```diff
@@ -77,7 +80,7 @@ def create_infrasim_conf():
     infrasim_conf = template.render(disks=disks, networks=networks)
     with open(INFRASIM_CONF, "w") as f:
         f.write(infrasim_conf)
-
+    os.system("ln -sf {} /etc/infrasim/infrasim.yml".format(INFRASIM_CONF))
```

Comment:
A comment would be helpful telling that convention is to update configuration file under /etc/*, while real hard copy is kept at workspace.